src/s2wasm.h

```diff
@@ -395,20 +403,59 @@ class S2WasmBuilder {
       skipWhitespace();
       if (debug) dump("process");
       if (!*s) break;
-      if (*s != '.') break;
+      if (*s == '.') {
+        s++;
+        if (match("text")) parseText();
+        else if (match("type")) parseType();
+        else if (match("weak") || match("hidden") || match("protected") || match("internal")) getStr(); // contents are in the content that follows
+        else if (match("imports")) skipImports();
+        else if (match("data")) {}
+        else if (match("ident")) skipToEOL();
+        else if (match("section")) parseToplevelSection();
+        else if (match("align") || match("p2align")) skipToEOL();
+        else if (match("globl")) parseGlobl();
+        else if (match("functype")) parseFuncType();
+        else parseObjectAlias(true);
+      } else parseObjectAlias(false);
+    }
+  }
+
+  // Aliases may (or not) be prefixed with period, but are consumed in process()
+  // Example: .foo = bla and foo = bla
```

Comment:
For aliases and labels, the period is just part of the name of the symbol and ideally should not be treated specially.

Actually probably we should handle object aliases in the `scan()` phase along with function aliases. Then I think `process()` wouldn't need to change.